Source/Core/VideoCommon/PixelShaderGen.cpp

```diff
@@ -495,19 +495,33 @@ void UpdateBoundingBoxBuffer(int2 min_pos, int2 max_pos) {{
 }}
 
 void UpdateBoundingBox(float2 rawpos) {{
-  // The rightmost shaded pixel is not included in the right bounding box register,
-  // such that width = right - left + 1. This has been verified on hardware.
-  int2 pos = int2(rawpos * cefbscale);
+  // This is kind of a hack. This is used to bias the coordinates "inwards" to produce
+  // more accurate (or at least less broken) results at higher resolutions. We also subtract
+  // an epsilon to avoid floating point precision error.
```

Comment:
Should it be said that being slightly smaller is better than being slightly oversized with examples of issues?

Reply:
I don't wanna make the comment too massive, but I can try to briefly explain I suppose.

Reply:
I just think having an example for future devs who may not be familiar with this can know why the decisions were made.

a simple "see: dolp.in/i/xxxx for info"